src/zenml/exceptions.py

```diff
@@ -168,6 +168,10 @@ class EntityExistsError(ZenMLBaseException):
     """Raised when trying to register an entity that already exists."""
 
 
+class EntityCreationError(ZenMLBaseException):
+    """Raised when failing to create an entity."""
+
+
```

Comment:
I bet we decided to stop creating custom exceptions. At least @stefannica always told me some for the core.

Reply:
I talked to @stefannica and we agreed it makes sense in this case.

Reply:
ok